problemsets-mat223/definitions.tex

```diff
@@ -113,29 +114,29 @@
 		\vec x=t\vec d+\vec p
 	\]
 	 is $\ell$ expressed in
-	\emph{vector form}. The vector $\vec d$ is called a
-	\emph{direction vector} for $\ell$.
+	\emph{vector form}\index{Line!vector form of}\index[definitions]{Line!vector form of}. The vector $\vec d$ is called a
+	\emph{direction vector}\index{Line!direction vector for}\index[definitions]{Line!direction vector for} for $\ell$.
 \end{SaveDefinition}
 
 \begin{SaveDefinition}[key=VectorFormofaPlane, title={Vector Form of a Plane}]
 	A plane $\mathcal P$ is written in
-	\emph{vector form} if it is expressed as
+	\emph{vector form}\index{Plane!vector form of}\index[definitions]{Plane!vector form of} if it is expressed as
 	\[
 		\vec x=t\vec d_{1} +s\vec d_{2}+\vec p
 	\]
 	 for some vectors $\vec d_{1}$ and $\vec d_{2}$ and point $\vec p$. That
 	is,
 	$\mathcal P = \{\vec x: \vec x= t\vec d_{1}+s\vec d_{2} +\vec p\text{ for
 	some }t,s\in\R \}$. The vectors $\vec d_{1}$ and $\vec d_{2}$ are called
-	\emph{direction vectors} for $\mathcal P$.
+	\emph{direction vectors}\index{Plane!direction vector for}\index[definitions]{Plane!direction vector for} for $\mathcal P$.
 \end{SaveDefinition}
 
 \begin{SaveDefinition}[key=Span, title={Span}]
 	The
-	\emph{span} of a set of vectors $V$ is the set of all linear
+	\emph{span}\index{Span}\index[definitions]{Span} of a set of vectors $V$ is the set of all linear
 	combinations of vectors in $V$. That is,
 	\[
-		\Span V = \Set{\vec v \given \vec v=\alpha_1\vec v_1+\alpha_2\vec
+		\Span V\index[symbols]{$\Span V$} = \Set{\vec v \given \vec v=\alpha_1\vec v_1+\alpha_2\vec
```

Comment:
Don't put `\index` in math mode. It's okay if it doesn't appear *right* next to the item. After all, it's only giving page references.